### PR TITLE
[AO] Propagate [x-request-id] header with the file transfer requests, do not log upscan download urls

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: app/uk/gov/hmrc/traderservices/controllers/FileTransferFlow.scala [hunk 1]
```diff
@@ -45,6 +45,7 @@ import scala.concurrent.duration.FiniteDuration
 import java.nio.charset.StandardCharsets
 import java.io.StringWriter
 import java.io.PrintWriter
+import java.util.UUID
```
Comment (crosslandwa) [inline app/uk/gov/hmrc/traderservices/controllers/FileTransferFlow.scala:48]: OBSERVATION: Unused import

File: app/uk/gov/hmrc/traderservices/models/TraderServicesFileTransferRequest.scala [hunk 2]
```diff
@@ -30,7 +30,8 @@ case class TraderServicesFileTransferRequest(
   fileMimeType: String,
   batchSize: Int,
   batchCount: Int,
-  correlationId: Option[String] = None
+  correlationId: Option[String] = None,
+  requestId: Option[String] = None
```
Comment (crosslandwa) [inline app/uk/gov/hmrc/traderservices/models/TraderServicesFileTransferRequest.scala:34]: OBSERVATION: Might be worth a comment explaining that these values are not expected to be supplied in the incoming JSON (that is deserialized into a `TraderServicesFileTransferRequest` instance)

**Review by crosslandwa — APPROVED** [review]
LGTM

File: app/uk/gov/hmrc/traderservices/controllers/FileTransferFlow.scala (hunk 1 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/controllers/FileTransferFlow.scala:48]: happily no fees nor VAT on scala imports!

**Pull request merged**